socorro/external/postgresql/fakedata.py

```diff
@@ -392,6 +392,130 @@ def daily_builds(self, fragment, channel, days=None):
+                        user_comments = None
+                        # if there is an email, always include a comment
+                        if email:
+                            user_comments = weighted_choice([
+                                (x, self.comments[x])
+                                for x in self.comments
+                            ])
+                        app_notes = ''
+                        distributor = ''
+                        distributor_version = ''
+                        topmost_filenames = ''
+                        addons_checked = 'f'
+                        flash_version = weighted_choice([
+                            (x, self.flash_versions[x])
+                            for x in self.flash_versions])
+                        hangid = ''
+                        process_type = weighted_choice([
+                            (x, self.process_types[x])
+                            for x in self.process_types])
+
+                        for buildid in buildids:
+                            row = [str(count),
+                                   client_crash_date,
+                                   date_processed,
+                                   self.generate_crashid(timestamp),
+                                   product,
+                                   number,
+                                   buildid,
+                                   signature,
+                                   url,
+                                   install_age,
+                                   last_crash,
+                                   uptime,
+                                   cpu_name,
+                                   cpu_info,
+                                   reason,
+                                   address,
+                                   os_name,
+                                   os_version,
+                                   email,
+                                   user_id,
+                                   started_datetime,
+                                   completed_datetime,
+                                   success,
+                                   truncated,
+                                   processor_notes,
+                                   user_comments,
+                                   app_notes,
+                                   distributor,
+                                   distributor_version,
+                                   topmost_filenames,
+                                   addons_checked,
+                                   flash_version,
+                                   hangid,
+                                   process_type,
+                                   channel_name,
+                                   product_guid,
+                                   exploitability,
+                                   channel_name]
+
+                            yield row
+                            count += 1
+
```

Comment:
Is this thing different from what it was when it belonged to the `Products` class?